include/openthread/link.h

```diff
@@ -1073,6 +1075,115 @@ bool otLinkIsPromiscuous(otInstance *aInstance);
  */
 otError otLinkSetPromiscuous(otInstance *aInstance, bool aPromiscuous);
 
+/**
+ * This function gets the CSL accuracy.
+ *
+ * @param[in]  aInstance      A pointer to an OpenThread instance.
+ *
+ * @returns The CSL accuracy in ppm.
+ *
+ */
+uint8_t otLinkCslGetAccuracy(otInstance *aInstance);
+
+/**
+ * This function sets the CSL accurary.
+ *
+ * @param[in]  aInstance      A pointer to an OpenThread instance.
+ * @param[in]  aAccurray      The accuracy of the clock that is used by the device, in units of ppm.
+ *
+ * @retval OT_ERROR_NONE           Successfully get the CSL parameters.
+ * @retval OT_ERROR_INVALID_ARGS   Invalid value for @p aAccuracy.
+ * @retval OT_ERROR_INVALID_STATE  Invalid state to change this parameter.
+ *
+ */
+otError otLinkCslSetAccuracy(otInstance *aInstance, uint8_t aAccuracy);
+
+/**
+ * This function gets the CSL channel.
+ *
+ * @param[in]  aInstance      A pointer to an OpenThread instance.
+ *
+ * @returns The CSL channel.
+ *
+ */
+uint8_t otLinkCslGetChannel(otInstance *aInstance);
+
+/**
+ * This function sets the CSL channel.
+ *
+ * @param[in]  aInstance      A pointer to an OpenThread instance.
+ * @param[in]  aChannel       The CSL sample channel.
+ *
+ * @retval OT_ERROR_NONE           Successfully get the CSL parameters.
+ * @retval OT_ERROR_INVALID_ARGS   Invalid @p aChannel.
+ * @retval OT_ERROR_INVALID_STATE  Invalid state to change this parameter.
+ *
+ */
+otError otLinkCslSetChannel(otInstance *aInstance, uint8_t aChannel);
+
+/**
+ * This function gets the CSL period.
+ *
+ * @param[in]  aInstance      A pointer to an OpenThread instance.
+ * @param[in]  aPeriod        The CSL period.
+ *
+ * @returns The CSL period in units of 10 symbols.
+ *
+ */
+uint16_t otLinkCslGetPeriod(otInstance *aInstance);
+
+/**
+ * This function sets the CSL period.
+ *
+ * @param[in]  aInstance      A pointer to an OpenThread instance.
+ * @param[in]  aPeriod        The CSL period in units of 10 symbols.
+ *
+ * @retval OT_ERROR_NONE           Successfully set the CSL period.
+ * @retval OT_ERROR_INVALID_ARGS   Invalid @p aPeriod.
```

Comment:
Invalid CSL period, not a pointer

Reply:
I think `@p` means parameter. I agree to change to `CSL period` anyway.

Reply:
Oh, my fault. Thanks for pointing this out.